ovirt_imageio/client/_upload.py

```diff
@@ -68,7 +68,9 @@ def register(parser):
 
 
 def upload_disk(args):
-    with _ui.ProgressBar(phase="inspecting image") as progress:
+    pb_format = _ui.FORMAT_JSON if args.json else None
```

Comment:
We should really have 2 format constants instead of using None, and maybe the argument
should not be boolean but string. The default can be "text" or "online". The user can set value "json":

    --format json

Will use json format. If not specified args.format will be "text". And we can just
pass arg.format to the call:

    with _ui.ProgressBar(
            phase="inspecting image", format=args.format) as progress: